feedback-suggestion/src/app/endpoints/FeedbackSuggestionRequest.py

```diff
@@ -110,3 +110,7 @@ def parse_lines(self):
 @router.get("/feedback_suggestions/feedbacks")
 def get_feedbacks():
     return db.fetch_feedbacks()
+
+@router.get("/feedback_suggestions/feedbacks/{exercise_id}")
+def get_feedbacks_for_exercise(exercise_id):
+    return db.fetch_feedbacks_for_exercise(exercise_id)
```

Comment:
```suggestion
    return db.fetch_feedbacks_by_exercise_id(exercise_id)
```

Wrong function name is used here